package.json

```diff
@@ -13,13 +13,13 @@
   "homepage": "https://thelounge.github.io/",
   "scripts": {
     "start": "node index",
-    "build": "npm run build:font-awesome && npm run build:grunt && npm run build:handlebars",
+    "build": "npm-run-all build:font-awesome build:grunt build:handlebars",
```

Comment:
Sorry to be that guy, but I'd really be more comfortable keeping `&&` here. Reason is that the `build` script (and associated `prepublish`) is the most critical script of all as it is run by Travis CI when we publish, and releases cannot be modified or removed once published.

I'm with keeping `test` and `lint` short and clean because it's just a matter of convenience in dev, but we've all been surprised by the over-engineering of the `npm-run-all` package, and if building works perfectly fine with `&&`, we should stick to that. Adding a dependency for this step is another point of failure we loosely control.

Does this make sense?